sdk/src/accounts/webSocketDriftClientAccountSubscriber.ts

```diff
@@ -75,6 +76,19 @@ export class WebSocketDriftClientAccountSubscriber
 			this.subscriptionPromiseResolver = res;
 		});
 
+		// If none of markets/oracles set, fetch em all
+		if (
+			this.perpMarketIndexes.length === 0 &&
+			this.spotMarketIndexes.length === 0 &&
+			this.oracleInfos.length === 0
```

Comment:
so now if a user wants to create a minimal viable driftClient, they'd have to pass in at least 1 of these in order to prevent unnecessary RPC calls?

i do find it sometimes useful to pass in nothing to drifClient if i just want to do something simple (place an order, read a user, ...)

Reply:
it was already using env to generate all the markets/oracle if they weren't explicitly passed. should i add a new param to the config to turn this off?